samples/demo/build.gradle

```
@@ -2,4 +2,6 @@ apply plugin: 'org.jetbrains.kotlin.jvm'
 apply plugin: 'com.jakewharton.mosaic'
 apply plugin: 'application'
 
-mainClassName = 'example.DemoKt'
+application {
+	mainClass = 'example.DemoKt'
+}
```

if you want to keep single-line, this should be an alternative syntax:
```suggestion
application.mainClass = 'example.DemoKt'
```